src/middleware/identity.rs

```diff
@@ -119,13 +118,35 @@ impl Identity {
             id.changed = true;
         }
     }
+
+    fn get_identity(extensions: &Extensions) -> Option<String> {
+        if let Some(id) = extensions.get::<IdentityItem>() {
+            id.id.clone()
+        } else {
+            None
+        }
+    }
 }
 
 struct IdentityItem {
     id: Option<String>,
     changed: bool,
 }
 
+/// Helper trait that allows to get Identity
```

Comment:
need more docs. add note that it could be used in middleware but identity policy must be set before any other middleware that needs identity